web/viewer-snippet-mozPrintCallback-polyfill.html

```diff
@@ -1,79 +1,45 @@
-<div id="mozPrintCallback-shim" hidden>
+<div id="mozPrintCallback-shim" class="container" hidden>
   <style>
-@media print {
-  #printContainer div {
-    page-break-after: always;
-    page-break-inside: avoid;
-  }
-}
+    @media print {
+      #printContainer div {
+        page-break-after: always;
+        page-break-inside: avoid;
+      }
+    }
   </style>
   <style scoped>
```

Comment:
`<style scoped>` only works in Firefox. Please insert the explicit `#mozPrintCallback-shim` selector back before each selector.